.github/workflows/build_assets.yml

```diff
@@ -21,23 +24,21 @@ jobs:
             TARGET: macos
             # currently, wrapt pulls the arm64 version instead of the universal one, so the below is a hack
             CMD_REQS: >
-              mkdir -p pip-packages && cd pip-packages && pip wheel --no-cache-dir --no-binary tree_sitter,ijson,charset_normalizer,PyYAML .. &&
+              mkdir -p pip-packages && cd pip-packages && pip wheel --no-cache-dir --no-binary ijson,charset_normalizer,PyYAML .. &&
               rm $(ls | grep wrapt) && pip download wrapt --platform=universal2 --only-binary=:all: && pip install $(ls | grep wrapt) --force-reinstall && cd .. &&
               pip install --no-deps --no-index --find-links=pip-packages pip-packages/*
             CMD_BUILD: >
-              STATICCODECOV_LIB_PATH=$(find build/ -maxdepth 1 -type d -name 'lib.*' -print -quit | xargs -I {} sh -c "find {} -type f -name 'staticcodecov*' -print -quit | sed 's|^./||'") &&
-              pyinstaller --add-binary ${STATICCODECOV_LIB_PATH}:. --copy-metadata codecov-cli --hidden-import staticcodecov_languages --target-arch universal2 -F codecov_cli/main.py &&
+              pyinstaller --copy-metadata codecov-cli --target-arch universal2 -F codecov_cli/main.py &&
               mv dist/main dist/codecovcli_macos &&
               lipo -archs dist/codecovcli_macos | grep 'x86_64 arm64'
             OUT_FILE_NAME: codecovcli_macos
             ASSET_MIME: application/octet-stream
-          - os: ubuntu-20.04
+          - os: ubuntu-latest
```

Comment:
I know we used `20.04` specifically for some reason, I probably would change this to `22.04` instead